modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -18,54 +31,69 @@ The following table contains examples of the URLs you need to supply, depending
-  *** For Salesforce, this value is `Bearer ${api_token}`, where `api_token` is an API token created through Salesforce.
-** *Client Request Timeout (seconds)*: The time elapsed before a client provider request times out. The minimum value is five seconds. The maximum value is 20 seconds.
 [NOTE]
 Anypoint Platform makes an HTTP call to the configured OIDC endpoints, and if there is no response after five seconds, the HTTP call times out.
-* *Token Introspection Client*
- ** *Client ID*: The client ID for an existing client in your IdP capable of introspection of _all_ tokens from all clients.
+.. Select *Advanced Settings* for these options:
+* *Disable server certificate validation*
++
+Disables server certificate validation if your OpenID client management instance presents a self-signed certificate, or one signed by an internal certificate authority.
+* *Enable client deletion in Anypoint Platform*
++
+Enables deletion of clients created with this integration.
+* *Enable client deletion and updates in IdP*
++
+To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform to `{clientRegistrationUrl}/{clientID}`. The `clientRegistrationUrl` is the value you configured in *Client Registration URL*. For example: `+https://identity.example.com/oauth2/connect/register/6779ef20e75817b79602+`. 
+. In *Authentication Method*, complete the information for the type of authentication to use for this client provider:
+* *Authorization Header*
++
+The HTTP authorization header provides a static token that doesn't expire to authenticate with a server. This header is required if the provider restricts registration requests to authorized clients. If you prefer to use short-lived bearer access tokens, use the OAuth 2.0 private key option instead. 
++
+[%autowidth.spread]
+|===
+|Client Provider | Value
+| Okta | `SSWS ${api_token}`, where `api_token` is an API token created through Okta.
+| ForgeRock | `Bearer ${api_token}`, where `api_token` is an API token created through ForgeRock.
+| Salesforce | `Bearer ${api_token}`, where `api_token` is an API token created through Salesforce.
+|===
+.. Click *Use OAuth2 private key jwt* to use a private key instead of and authorization header, and provide these values:
+* *Token Client ID*
++
+This is the JWT (JSON Web Token) that contains information about the authenticated user, used by OIDC to verify that the user is who they claim to be. 
+* *Token Client Private Key*
++
+This is the secret cryptographic key used by the client to authenticate itself to a server when requesting access tokens.
+* *Token Client API Scopes*
++
+The client API scopes define the specific permissions granted to an application when requesting access to a resource.
```

Comment:
`..an application..` The application is actually the token client above. Anypoint uses this token client with the scopes to register new clients.